### ISSUE TITLE
Usage of random_demo

### ISSUE THREAD

**Issue by keep9oing** [PettingZoo-Team/PettingZoo#344]
Hi, thanks for your environments,

I just have a question about using random_demo util

```python
from pettingzoo.magent import gather_v2
from pettingzoo.utils import random_demo

env = gather_v2.env()
random_demo(env, render=True, episodes=30)
```

This only shows the initial scene.
Other magent environments work like this too.
How can I use it correctly?

**Comment by weepingwillowben** [comment]
Sorry, it looks like the `random_demo` is broken in the latest release of pettingzoo (1.6.0). It is fixed on master. 

Also note that gather_v2 will render very slowly because it is such a large environment. I recommend trying to switch to battle_v2 to test if it will work.

**Comment by weepingwillowben** [comment]
This fix has been released in Pettingzoo 1.6.1
